scss/footer.scss

```diff
@@ -0,0 +1,29 @@
+footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 40px 0 10px 0;
```

Comment:
Try to apply this:
<img width="946" alt="captura de pantalla 2018-06-01 a la s 15 56 07" src="https://user-images.githubusercontent.com/36163179/40860699-57671d7a-65b4-11e8-9e56-035cc019a2bd.png">